src/main/components/DataTree.jsx

```diff
@@ -1,34 +1,29 @@
 import React, { useState, useEffect } from 'react';
 import ReactJson from 'react-json-view';
-import JsonTree from '../lib/treeView/jsonTree';
 import '../lib/treeView/jsonTree.css';
 
 const DataTree = (props) => {
-  const { treeCount, data, options } = props;
+  const { treeCount, data, options, saveUpdatedTree } = props;
```

Comment:
I _think_ @Donaldblanc’s liner is going to want this split into separate lines, e.g. 
```suggestion
  const { 
  treeCount, data, options, saveUpdatedTree 
  } = props;
```